src/unity/python/turicreate/toolkits/classifier/logistic_classifier.py

```diff
@@ -621,10 +622,11 @@ class as a vector. The probability of the first class (sorted
         >>> class_predictions = model.predict(data, output_type='class')
 
         """
-
-        return super(_Classifier, self).predict(dataset,
-                                                output_type=output_type,
-                                                missing_value_action=missing_value_action)
+        if self.num_classes >2 and output_type=='probability':
+            raise ToolkitError("Not for multiclass")
+        return super(_Classifier, self).predict(dataset,output_type=output_type,missing_value_action=missing_value_action)
```

Comment:
Nit: I don't think you need to touch this line. Leaving it alone avoids cluttering the git blame/history for this line

Reply:
Okay